be/src/exec/pipeline/audit_statistics_reporter.cpp

```diff
@@ -17,18 +17,28 @@
 #include <thrift/Thrift.h>
 #include <thrift/protocol/TDebugProtocol.h>
 
-#include "agent/master_info.h"
 #include "gen_cpp/FrontendService_types.h"
 #include "runtime/client_cache.h"
 #include "runtime/exec_env.h"
-#include "service/backend_options.h"
 
 namespace starrocks::pipeline {
 
 using apache::thrift::TException;
 using apache::thrift::TProcessor;
 using apache::thrift::transport::TTransportException;
 
+AuditStatisticsReporter::AuditStatisticsReporter() {
+    auto status = ThreadPoolBuilder("audit_report") // exec state reporter
+                          .set_min_threads(1)
```

Comment:
In high concurrent, is two threads enough? will the audit report block the insert task finish?

Reply:
it's the same size of the exec state reporter. And for the high concurrent situations, this may block the insert task to finished.